src/bci_build/package/__init__.py

```diff
@@ -930,6 +933,11 @@ def pkg_filter_func(p: str | Package) -> bool:
 
             return pkg_filter_func
 
+        def pkg_listing_func(pkg: Package) -> str:
+            if pkg.replaces:
+                return f'<package name="{pkg}" replaces="rpm"/>'
```

Comment:
The `replaces=` attribute has been deleted in schemaversion 6.7:
```
    Add XSL stylesheet to convert from v6.6 to v6.7
    
    Delete the following obsolete and unhandled attributes and sections
    
    * <image ... kiwirevision="..."/>
    * <type ... checkprebuilt="..."/>
    * <type ... fsnocheck="..."/>
    * <repository ... status="..."/>
    * <repository ... prefer-license="..."/>
    * <package replaces="..."/>
    * <defaultprebuilt>
    * <defaultdestination>
    * <defaultroot>
    * <partitioner>
    * <rpm-force>
```

Reply:
This has been fixed now.